### PR TITLE
feat: apps `--reveal` flag to expose secrets

### PR DESCRIPTION
**Pull request by jfatta**
### some examples
`./auth0 apps show --format=json`
image

`./auth0 apps show`
image

`./auth0 apps show --reveal`
image

`./auth0 apps show --reveal --format=json`
image

### CODE REVIEW

**Review by Widcket — CHANGES_REQUESTED** [review]
Thanks, this is a good start. It still needs to strip the secret from the JSON, and the keys as well.

File: internal/cli/apps_test.go [hunk 1]
```diff
@@ -12,46 +12,77 @@ import (
...
-	clientAPI.EXPECT().
-		List(gomock.Any()).
-		Return(&management.ClientList{
-			Clients: []*management.Client{
-				{
-					Name:      auth0.String("some-name"),
-					ClientID:  auth0.String("some-id"),
-					Callbacks: stringToInterfaceSlice([]string{"http://localhost"}),
-				},
+	tests := []struct {
```
Comment (Widcket) [inline internal/cli/apps_test.go:15]: 👏🏼 👏🏼 👏🏼

**Comment by jfatta** [comment]
removing signing keys too

image

File: internal/cli/apps.go [hunk 2]
```diff
@@ -179,6 +186,7 @@ func useAppCmd(cli *cli) *cobra.Command {
 }
 
 func listAppsCmd(cli *cli) *cobra.Command {
+	var inputs struct{ Reveal bool }
```
Comment (cyx) [inline internal/cli/apps.go:189]:
```suggestion
	var inputs struct{
		Reveal bool
	}
```

While it's more lines, consistency is king.

**Review by cyx — APPROVED** [review]
Awesome 👍

File: internal/cli/apps.go [hunk 3]
```diff
@@ -111,6 +111,13 @@ var (
 		Help:       "List of grant types supported for this application. Can include code, implicit, refresh-token, credentials, password, password-realm, mfa-oob, mfa-otp, mfa-recovery-code, and device-code.",
 		IsRequired: false,
 	}
+	reveal = Flag{
+		Name:       "Reveal",
+		LongForm:   "reveal",
+		ShortForm:  "r",
+		Help:       "Display the Client Secret as part of the command output.",
+		IsRequired: false,
```
Comment (Widcket) [inline internal/cli/apps.go:119]:
```suggestion
```
This is the default value

**Review by Widcket — APPROVED** [review]
Thanks for cleaning up the display KeyValues methods!

**Pull request merged**